scripts/validate/validate_bindings.sh

```diff
@@ -2,8 +2,6 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Comment:
Is this needed anymore?

Reply:
@mikebin as the code is today, `DIR` is used in the next line.

Reply:
Oops, missed that. Sorry for the noise.